app/models/guide_republisher.rb

```diff
@@ -12,7 +12,7 @@ def republish
 
       publishing_api.put_content(guide.content_id, guide_for_publication.content_payload)
```

Comment:
Should this have had 'minor' moved to here, or am I not understanding this change?